cabinet/cabinet.py

```diff
@@ -56,6 +56,20 @@ def get_all(self):
     def get(self, name):
         return self._vault.get(name)
 
+    def get_by_tags(self, tags=[]):
+        """
+        Recover all the items that contains the given tags.
+
+        @param tags: A list of tags
```

Comment:
Use `:param` instead of `@param`.
This is not wrong, just inconsistent with other docstrings across the module, we can change to a different docstring format, but for now we are using Sphinx.